SPECS/application-gateway-kubernetes-ingress/application-gateway-kubernetes-ingress.spec

```
@@ -24,9 +24,13 @@ Source0:        %{name}-%{version}.tar.gz
 #           -cf %%{name}-%%{version}-vendor.tar.gz vendor
 #
 Source1:        %{name}-%{version}-vendor.tar.gz
+
+# patches for vendored code >= 1000
 # If upstream ever upgrades client_goland to 1.11.1, we can get rid of this patch.
-Patch0:         CVE-2022-21698.patch
-Patch1:         CVE-2023-44487.patch
+Patch1000:      CVE-2022-21698.patch
+Patch1001:      CVE-2023-44487.patch
+Patch1002:      CVE-2021-44716.patch
```

There might have been a sequencing issue, I removed the >= 1000 numbering on most golang packages.
```suggestion
Patch0:         CVE-2022-21698.patch
Patch1:         CVE-2023-44487.patch
Patch2:         CVE-2021-44716.patch
```